docs/book/features/block_volume.md

```diff
@@ -375,3 +376,67 @@ Cluster
             VolumeAttributes:      type=vSphere CNS Block Volume
         Events:                <none>
     ```
+
+### Statically Provision a Block Volume in Tanzu Kubernetes Grid Cluster<a id="static_volume_provisioning_tkg"></a>
+
+You can statically create a block volume in a Tanzu Kubernetes Grid (TKG) Cluster using an unused PersistentVolumeClaim (PVC) from the Project Pacific cluster only if the latter satisfies the following conditions:
+
+- It is present in the same namespace as the TKG Cluster
+
+- It is associated with the same `StorageClass` as the default storage class in the TKG Cluster
+
+- Not attached to a Pod in the Project Pacific Cluster
+
+Steps to create a static PVC in a TKG Cluster are as follows:
+
+1. Note the name of the PVC in the Project Pacific cluster
+
+2. Create a PersistentVolume with the `storageClassName` set to the default storage class of the TKG Cluster and the `volumeHandle` pointing to the value retrieved in step 1.
+
+    ```yaml
+      apiVersion: v1
+      kind: PersistentVolume
+      metadata:
+        name: static-tkg-block-pv
+        annotations:
+          pv.kubernetes.io/provisioned-by: csi.vsphere.vmware.com
+      spec:
+        storageClassName: "gc-storage-profile"  # Replace this value with the default storage class in your TKG Cluster
+        capacity:
+          storage: 2Gi
+        accessModes:
+          - ReadWriteOnce
+        persistentVolumeReclaimPolicy: Delete
+        csi:
+          driver: "csi.vsphere.vmware.com"
+          volumeAttributes:
+            type: "vSphere CNS Block Volume"
+          "volumeHandle": "project-pacific-block-pvc-name"  # Mention the PVC name from Project Pacific cluster here
+    ```
+
+3. Create a PVC to match the PV object created above. Also set the `storageClassName` to the same value as above.
+
+    ```yaml
+      kind: PersistentVolumeClaim
+      apiVersion: v1
+      metadata:
+        name: static-tkg-block-pvc
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 2Gi
```

Comment:
this should match with PV, not sure if we should mention this or not

Reply:
Not required, this is not vSphere specific.